workspace_tools/targets.py

```diff
@@ -842,6 +842,23 @@ def __init__(self):
         NRF51822.__init__(self)
         self.extra_labels = ['NORDIC', 'MCU_NRF51822', 'MCU_NORDIC_16K']
         self.macros = ['TARGET_NRF51822']
+		
+class DELTA_DFCM_NNN40(NRF51822):
+    def __init__(self):
+        NRF51822.__init__(self)
+        self.core = "Cortex-M0"
+        self.extra_labels = ['NORDIC', 'MCU_NRF51822', 'MCU_NORDIC_16K']
+        self.macros = ['TARGET_NRF51822']
+        self.supported_toolchains = "GCC_ARM"
```

Comment:
Just add a new commit to your branch, will be reflected here.

To add support for another toolchain - add it to targets script file, plus cmsis files (which are there for other targets so reuse those if they match), and add exporters templates (workspace_tools/exporters/). There's a toolchain script file where is an array of supported targets - add yours there, plus .tmpl file, which again is for other targets with the same MCU, so it's matter of copy-paste. This can be sent as another pull request.